demo/geo_json/world_country_iso_mapping.json

```diff
@@ -0,0 +1,175 @@
+{
+  "AE": ["United Arab Emirates"],
```

Comment:
since we no longer care about variants, we could simplify this file and the `usa_sates_mapping.json` by inverting it directly rather than using `inverseMapping`

`"United Arab Emirates": "AE" `

Reply:
I'd rather keep it as it. It improves a bit the usability of geo charts, and we need a mapping anyway in demo, so the only read cost of having this is the 5 lines of the `inverseMapping` function.

Reply:
ok ok